tests/integration/github/test_generic_commands.py

```diff
@@ -216,3 +216,7 @@ def test_is_private(self):
 
     def test_is_not_private(self):
         assert not self.ogr_project.is_private()
+
+    def test_delete(self):
+        project = self.service.get_project(repo="temp-file", namespace="shreyaspapi")
```

Comment:
Just a nit-pick -- what about using the same name as for others?

Reply:
Done👍